actiontext/lib/action_text/fixture_file_helpers.rb

```diff
@@ -0,0 +1,12 @@
+module ActionText
+  module FixtureFileHelpers
+    def action_text_attachment(fixture_set_name, label)
```

Comment:
Related to https://github.com/rails/rails/pull/40289/files#r495132398, I had considered making this its own `ActiveRecord::FixtureSet` clone, and forcing callers to fully qualify their calls as if they were calling something like `identify`:

```yaml
body: "<p><%= ActionText::FixtureSet.action_text_attachment %></p>"
```

That would avoid the auto-loading issues, but could be verbose. If this method is prefered, we could drop the `action_text` prefix and instead declare `ActionText::FixutreSet.attachment`, which isn't half-bad.

Reply:
After more consideration, I've pushed up https://github.com/rails/rails/pull/40289/commits/7dfd9773b95351655ed44b5ea2648ddf2e58c9d2 to propose those changes instead.